### DISCUSSION TITLE
usage inside typescript playground

### DISCUSSION THREAD

**Discussion by o-az** [jsr-io/jsr#710] [Q&A] [answered]
I tried the following

```ts
import { camelCase } from "@jsr/luca__cases";

console.info(camelCase)
```

https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAbzgYwIYgKYBsDCqDOGcAvnAGZQQhwBEAAgFb5QD0WArmgPpdqH40A3AChhyCADt8ELBgB0wCWQgAKNJlwEMASiA

result:

```
Cannot find module '@jsr/luca__cases' or its corresponding type declarations.
```

**Comment by marvinhagemeister** [comment, marked as answer]
I don't think the TypeScript playground supports pulling from registries other than npm itself.
